appveyor.yml

```diff
@@ -10,10 +10,10 @@ environment:
 
   matrix:
     # Since appveyor is quite slow, we only use a single configuration
-    - PYTHON: "3.5"
+    - PYTHON: "3.6"
       ARCH: "64"
-      NUMPY: "1.12.1"
-      PANDAS: "0.19.2"
+      NUMPY: "1.15"
```

Comment:
Maybe try exact versions like 1.15.2 and 0.23.4?

Reply:
Sadly not:

```
PackageNotFoundError: Package not found: '' Packages missing in current win-64 channels: 
  - numpy 1.15.2
  - pandas 0.23.4
```